plugins/generator-1.20.1/forge-1.20.1/procedures/entity_isaggressive.java.ftl

```diff
@@ -0,0 +1 @@
+(${input$entity} instanceof Mob _mobEnt ? _mobEnt.isAggressive():false)
```

Comment:
change `_mobEnt` to `_mobEnt${cbi}` to work correctly with nested procedure statements

Reply:
Tbh I just copied the code from the isleashed procedure. Either way, fixed.